manager/config/config.go

```diff
@@ -338,6 +338,9 @@ type SyncPeersConfig struct {
 
 	// Timeout is the timeout for syncing peers information from the single scheduler.
 	Timeout time.Duration `yaml:"timeout" mapstructure:"timeout"`
+
+	// BatchSize is the batch size when operating gorm.
+	BatchSize int `yaml:"batchSize" mapstructure:"batchSize"`
```

Comment:
where is the usage of this config?

Reply:
used at https://github.com/CormickKneey/Dragonfly2/blob/74ea568a9b14de706f7a0d68e8c1f633e4f771d6/manager/job/sync_peers.go#L248

for possible huge data or future optimization